src/app/layouts/PrivateLayout.tsx

```diff
@@ -1,17 +1,41 @@
 import type { FC } from 'react';
 import { Navigate, Outlet } from 'react-router';
+import { useQuery } from '@tanstack/react-query';
 import { cookieManager } from '@shared/lib/cookieManager';
 import { setupAuthInterceptors } from '@features/auth/model/auth.interceptor';
 import { axiosInstance } from '@shared/api/axios';
 import { PrivateHeader } from '@widgets/PrivateHeader';
 import { Sidebar } from '@widgets/Sidebar';
 import { BackToButton } from '@widgets/Sidebar/ui/BackToButton';
+import { currenciesApi } from '@entities/currencies';
+import { categoriesApi } from '@entities/categories';
+import { useStore } from '@shared/lib/useStore';
+
+setupAuthInterceptors(axiosInstance);
 
 export const PrivateLayout: FC = () => {
   const token = cookieManager.getCookie('accessToken');
+  const { categoriesStore, currenciesStore } = useStore();
 
-  setupAuthInterceptors(axiosInstance);
+  useQuery({
```

Comment:
We shouldn't use useQuery in components, need to add business logic in 
`/entities/currencies/models/useCurrenciesQuery`
Page shouldn't know about this business logic.
```suggestion
  useCurrenciesQuery();
  useCategoriesQuery();
```